trin-state/src/jsonrpc.rs

```diff
@@ -42,7 +50,43 @@ impl StateRequestHandler {
                 recursive_find_nodes(network, node_id).await
             }
             StateEndpoint::DataRadius => radius(network),
-            _ => Err("Not implemented".to_string()),
+            StateEndpoint::LocalContent(content_key) => local_content(network, content_key).await,
+            StateEndpoint::FindContent(enr, content_key) => {
+                find_content(network, enr, content_key).await
+            }
+            StateEndpoint::RecursiveFindContent(content_key) => {
+                recursive_find_content(network, content_key, /* is_trace= */ false).await
```

Comment:
Nice, didn't know we could inline comments this way 💡

Reply:
As I've mentioned in a previous pr, I don't love it. But, I guess in this case especially I see the usefulness. And creating a `let is_trace=false;` variable does seem overkill. So, maybe I'll have to change my mind about this

Reply:
They might look weird at first, but they grow on you.

They shouldn't be overused, but they have their use cases.